wrappers/golang/curves/bls12377/ntt_test.go

```diff
@@ -120,6 +120,7 @@ func TestECNtt(t *testing.T) {
 
 func TestNttDeviceAsync(t *testing.T) {
 	cfg := GetDefaultNttConfig()
+	initDomain(largestTestSize, cfg)
```

Comment:
Was this test consistently failing when running the entire test file or only when running this test?

Reply:
Consistently on running only this test, randomly on CI/running all the tests

Reply:
> Consistently on running only this test

This makes sense. All the other tests as well that rely on initDomain will have the same issue.

> randomly on CI/running all the tests

I guess the `go test` engine doesn't have a deterministic order for calling tests... i'll look into it but I think my suggestion [here](https://github.com/ingonyama-zk/icicle/pull/461#pullrequestreview-1984884792) is likely the best solution for now